packages/ui/src/components/organisms/ScrollList/ScrollList.tsx

```diff
@@ -106,33 +119,38 @@ const ScrollList = ({
   }
 
   return (
-    <FlatList
-      style={[styles.scrollView, style]}
-      data={data}
-      onRefresh={disabledRefresh ? null : updateData}
-      refreshing={loadingList}
-      horizontal={horizontal}
-      onEndReached={onEndReached}
-      ListFooterComponent={() => {
-        return (
-          <View style={[styles.footerText, styleFooter]}>
-            {moreLoading && <ActivityIndicator size="large" color="black" />}
-            {isListEnd && (
-              <Text>
-                {data == null || data?.length === 0
-                  ? translator != null
-                    ? translator('Base_NoData')
-                    : 'No data.'
-                  : translator != null
-                  ? translator('Base_NoMoreItems')
-                  : 'No more items.'}
-              </Text>
-            )}
-          </View>
-        );
-      }}
-      renderItem={renderItem}
-    />
+    <>
+      {Array.isArray(actionList) && actionList.length > 0 && (
+        <TopActions actionList={actionList} verticalActions={verticalActions} />
+      )}
```

Comment:
Please add animations on this to make it disappear when user scroll down and re-appear when user scroll up.